src/uucore/src/lib/features/signals.rs

```diff
@@ -341,6 +349,13 @@ pub fn signal_by_name_or_value(signal_name_or_value: &str) -> Option<usize> {
         .map(|s| s.value)
 }
 
+pub fn signal_name_by_value(signal_value: usize) -> Option<&'static str> {
+    ALL_SIGNALS
+        .iter()
+        .find(|signal| signal.value == signal_value)
+        .map(|signal| signal.name)
+}
```

Comment:
I'm not familiar with this code, but it seems that the indexes of the signals in the `ALL_SIGNALS` array correspond with their value. What is the reason that we can't just use `ALL_SIGNALS[signal_value]`? Is the order not guaranteed?

Reply:
That's true now that I added the `Exit` signal. I think we can change it so that the value is not explicitly stored.

Reply:
Right, that could be done another PR, I guess.